src/components/SingleEvent/Admin/PartyForm.js

```diff
@@ -187,6 +187,23 @@ class PartyForm extends Component {
     })
   }
 
+  componentDidMount() {
+    let klass = this
```

Comment:
Do we need this? Aren't you already using an arrow function inside the promise, so you shouldn't need to bind `this`

Reply:
fixed